homeassistant/components/vicare/strings.json

```diff
@@ -162,100 +162,112 @@
         "name": "DHW storage bottom temperature"
       },
       "hotwater_gas_consumption_today": {
-        "name": "DHW gas consumption today"
+        "name": "Gas consumption (DHW) today"
       },
       "hotwater_gas_consumption_heating_this_week": {
-        "name": "DHW gas consumption this week"
+        "name": "Gas consumption (DHW) this week"
       },
       "hotwater_gas_consumption_heating_this_month": {
-        "name": "DHW gas consumption this month"
+        "name": "Gas consumption (DHW) this month"
       },
       "hotwater_gas_consumption_heating_this_year": {
-        "name": "DHW gas consumption this year"
+        "name": "Gas consumption (DHW) this year"
       },
       "gas_consumption_heating_today": {
-        "name": "Heating gas consumption today"
+        "name": "Gas consumption (heating) today"
       },
       "gas_consumption_heating_this_week": {
-        "name": "Heating gas consumption this week"
+        "name": "Gas consumption (heating) this week"
       },
       "gas_consumption_heating_this_month": {
-        "name": "Heating gas consumption this month"
+        "name": "Gas consumption (heating) this month"
       },
       "gas_consumption_heating_this_year": {
-        "name": "Heating gas consumption this year"
+        "name": "Gas consumption (heating) this year"
       },
       "gas_consumption_fuelcell_today": {
-        "name": "Fuel cell gas consumption today"
+        "name": "Gas consumption (power) today"
       },
       "gas_consumption_fuelcell_this_week": {
-        "name": "Fuel cell gas consumption this week"
+        "name": "Gas consumption (power) this week"
       },
       "gas_consumption_fuelcell_this_month": {
-        "name": "Fuel cell gas consumption this month"
+        "name": "Gas consumption (power) this month"
       },
       "gas_consumption_fuelcell_this_year": {
-        "name": "Fuel cell gas consumption this year"
+        "name": "Gas consumption (power) this year"
       },
       "gas_consumption_total_today": {
-        "name": "Gas consumption today"
+        "name": "Gas consumption (total) today"
```

Comment:
Maybe we can omit `total`

Reply:
Also no need to change that, like all the "gas consumption entities".

The problem is with all the "energy" entities which should be "electricity" instead.